### PR TITLE
CUMULUS-2558 -- Fix execution overwrite bug

### PR DESCRIPTION
**Pull request by Jkovarik**
Executions were not saving the originalPayload value as the
sf-even-sqs-to-db-records write-execution code was populating keys for
upsert with nil values -- this results in *overwriting* values from
the original record, which is inconsistent with the dynamo
implementation

Addresses [CUMULUS-2558](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2558)

## Changes

Bugfix: 

Executions were not saving the original_payload value in postgres as the
sf-even-sqs-to-db-records write-execution code was populating keys for
upsert with nil values -- this results in *overwriting* values from
the original record, which is inconsistent with the dynamo
implementation

Integration coverage will come from the 'read executions RDS' feature branch workk, as we will start reading these records from the API. 

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [x] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -81,6 +93,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- **CUMULUS-2558**
+  - Fixed issue where executions original_payload would not be retained on successful execution
```
Comment (Jkovarik) [inline CHANGELOG.md:97]: This is the CL entry for this ticket.   The rest is re-wrap spam 😢

File: packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-execution.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ const test = require('ava');
 const cryptoRandomString = require('crypto-random-string');
 const sinon = require('sinon');
 const uuidv4 = require('uuid/v4');
+const { removeNilProperties } = require('@cumulus/common/util');
+
```
Comment (botanical) [inline packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-execution.js:8]: Remove newline

File: packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-execution.js [hunk 3]
```diff
@@ -4,6 +4,8 @@ const test = require('ava');
 const cryptoRandomString = require('crypto-random-string');
 const sinon = require('sinon');
 const uuidv4 = require('uuid/v4');
+const { removeNilProperties } = require('@cumulus/common/util');
```
Comment (botanical) [inline packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-execution.js:7]: Did you mean to use this import?

File: packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-execution.js (hunk 2 above)
Comment (Jkovarik) [reply packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-execution.js]: No, thanks for catching that. I was getting a bit too caviler with my cherry picking. 

6641a64b8

**Review by markdboyd — APPROVED** [review]

**Pull request merged**